### PR TITLE
Move test bots

### LINKED ISSUE #6000: Move tests bots in solution to a TestBots section
## Summary

We have several bots in the Tests folder that we use for local testing and development of framework features. These test bots get stale and bloat the solution over time. Long term, we would like to sanitize the test bots to a small number that we actually use for dev and also configure functional tests on them to ensure those bots always run and test product features. 

As part of this issue, we will just move these projects to a testbots folder in the file system and the VS solution. The test folder will only contain unit tests projects, no test bots there. 

The solution should look like this:

image

And files in the file system should look like this.

image

There should be a 1 to 1 mapping between the projects and directories in the libraries folder and projects and directories in the Tests folder.

### PR DESCRIPTION
**Pull request by mrivera-ms**
Fixes #6000 
This change moves the files in VS and the file system to have the folder structure match the virtual folders in the Solution.

image

### CODE REVIEW

File: TestBots/Microsoft.Bot.Builder.TestBot/Microsoft.Bot.Builder.TestBot.csproj [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <ProjectReference Include="..\..\libraries\integration\Microsoft.Bot.Builder.Integration.AspNet.Core\Microsoft.Bot.Builder.Integration.AspNet.Core.csproj" />
     <ProjectReference Include="..\..\libraries\Microsoft.Bot.Builder.AI.LUIS\Microsoft.Bot.Builder.AI.Luis.csproj" />
     <ProjectReference Include="..\..\libraries\Microsoft.Bot.Builder.Dialogs\Microsoft.Bot.Builder.Dialogs.csproj" />
-    <ProjectReference Include="..\Microsoft.Bot.Builder.TestBot.Shared\Microsoft.Bot.Builder.TestBot.Shared.csproj" />
+    <ProjectReference Include="..\..\tests\Microsoft.Bot.Builder.TestBot.Shared\Microsoft.Bot.Builder.TestBot.Shared.csproj" />
```
Comment (gabog) [inline TestBots/Microsoft.Bot.Builder.TestBot/Microsoft.Bot.Builder.TestBot.csproj:18]: I think this one (testbot.shared.csproj)is only used by test bots and can be moved into the testbots folder

File: TestBots/Directory.Build.props [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!-- Contains common properties that apply to projects under the Tests folder -->
```
Comment (gabog) [inline TestBots/Directory.Build.props:1]:
```suggestion
<!-- Contains common properties that apply to projects under the TestBots folder -->
```

**Review by gabog — COMMENTED** [review]
See my comments and let me know what you think.

Also, all the other folders are lowercase, do we want to use TestBots, testbots or testBots?

File: TestBots/Microsoft.Bot.Builder.TestBot/Microsoft.Bot.Builder.TestBot.csproj (hunk 1 above)
Comment (mrivera-ms) [reply TestBots/Microsoft.Bot.Builder.TestBot/Microsoft.Bot.Builder.TestBot.csproj]: Moved.

File: TestBots/Directory.Build.props (hunk 2 above)
Comment (mrivera-ms) [reply TestBots/Directory.Build.props]: Changed.

**Review by gabog — APPROVED** [review]

**Pull request merged**